test/powershell/Modules/Microsoft.PowerShell.Utility/JsonObject.Tests.ps1

```diff
@@ -1,46 +1,33 @@
 ﻿Describe 'Unit tests for JsonObject' -tags "CI" {
 
-    function ShouldThrow
-    {
-        param (
-            [Parameter(ValueFromPipeline = $true)]
-            $InputObject,
-            [Parameter(Position = 0)]
-            $ExpectedException
-        )
-
-        try
-        {
-            & $InputObject
-            throw "Should throw exception"
-        }
-        catch
-        {
-            $_.FullyQualifiedErrorId | should be $ExpectedException
-        }
-    }
+    $TestCasesForReturnHashTableParameter = @($true, $false)
 
     $validStrings = @(
-        @{ name = "empty"; str = "" }
-        @{ name = "spaces"; str = "  " }
-        @{ name = "object"; str = "{a:1}" }
+        @{ name = "empty";  str = "";      ReturnHashTable = $true  }
+        @{ name = "spaces"; str = "  ";    ReturnHashTable = $true  }
+        @{ name = "object"; str = "{a:1}"; ReturnHashTable = $true  }
+        @{ name = "empty";  str = "";      ReturnHashTable = $false }
+        @{ name = "spaces"; str = "  ";    ReturnHashTable = $false }
+        @{ name = "object"; str = "{a:1}"; ReturnHashTable = $false }
     )
```

Comment:
Please move to BeforeAll block.

Reply:
I am not sure what you exactly mean by that. I removed the redundant line about `$TestCasesForReturnHashTableParameter that was a leftover.

Reply:
Please see [sample](https://github.com/PowerShell/PowerShell/blob/309bf603f9aff9009dce7e725d42b2d4f99f58c9/test/powershell/Modules/Microsoft.PowerShell.Management/Clear-Item.Tests.ps1#L2) 
We should place ` $TestCasesForReturnHashTableParameter ` and `$validStrings ` in the BeforeAll block.